Code-the-Dream-School/dd-prac-team1-front
# Dev/shopping list add ingredients (PRA-203)

## One Line Description
implementing add new ingredient with backend API
## Requirements

_What this code should do_
1. add new ingredient and process when the usr try to add existing ingredient
## Notes
the next PR will be handle units unifications. The full responsiveness will be in the last PR.  
_Any additional info_

## Test Steps
1. try to add new ingredient. 
2. 2. if the ingredient is already exist you should see the toaster with this backend response. After you will scroll to the highlighted ingredient.  the highlighting will disappear after 3 sec
_Instructions on how to test the changes_

## UI/UX Outcome
<img width="809" alt="image" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/106414157/2ccc5e79-396e-4e70-b952-9c3fcd434a73">
<img width="807" alt="image" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/106414157/9b3a1006-2b92-472c-9bc3-85112ddef8ee">

_Attach a screenshot here_

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
